File: PDOO/p4/civitas_rb/lib/juegotexto.rb
require_relative 'vistatextual'
require_relative 'civitas_juego'
require_relative 'controlador'
require_relative 'dado'

module Civitas
  
class Juegotexto
  def self.main
    vista = Vistatextual.new
    
    nombres = Array.new
    for i in 1..4
      nombres.push("Jugador#{i}")
    end
    
    juego = Civitas_juego.new(nombres)
    
    Dado.instance.setdebug(true)
    
    controlador = Controlador.new(juego, vista)
    
    controlador.juega()
  end
end

Juegotexto.main


end

File: PDOO/PDOO/Java/Civitas/src/JuegoTexto/JuegoTexto.java

package juegoTexto;


import civitas.CivitasJuego;
import civitas.Dado;
import java.util.ArrayList;


public class JuegoTexto {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        
        VistaTextual vista = new VistaTextual();
        
        ArrayList<String> nombres = new ArrayList<>();
        nombres.add("Jugador1");
        nombres.add("Jugador2");
        nombres.add("Jugador3");
        nombres.add("Jugador4");
        CivitasJuego juego = new CivitasJuego(nombres);
        
        Dado.getInstance().setDebug(true);
        
        Controlador controlador = new Controlador(juego, vista);
        
        controlador.juega();
    }
    
}
